addons/html_editor/static/src/main/banner_plugin.js

```diff
@@ -137,10 +137,6 @@ export class BannerPlugin extends Plugin {
         this.dependencies.selection.setCursorEnd(
             bannerElement.querySelector(`.o_editor_banner_content > ${baseContainer.tagName}`)
         );
-        // Remove the old element.
-        if (bannerElement.nextSibling?.nodeName === blockEl.nodeName) {
-            bannerElement.nextSibling.remove();
-        }
```

Comment:
@Zynton You added these lines in https://github.com/odoo/odoo/pull/234185/changes/06b5513fab1eb4b8c3fe4fbad09b77ab9e76e751#diff-efb554f32fa0897c9545dbd9f472fe61f00a3e484f27e4be4b079e8e25588e89R139 but I don't understand why. Ok, you don't need automatic paragraphs after a banner anymore, fine, but why _remove_ existing nodes?

It seems like you did something similar in https://github.com/odoo/odoo/pull/234185/changes/674eed4bf51ba4b7302000a64aa08468298064c3#diff-4547ce2119bfa4f29ea7766ae74e93cd34d09e9ff1602c0b23ee45735ecec0cbR129 as well so I'm a bit worried.